src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/AdditionalInformation.kt

```diff
@@ -15,9 +15,9 @@ sealed class AdditionalInformation {
   data class PrisonerReceivedAdditionalInformation(
     val nomsNumber: String,
     val reason: Reason,
-    val details: String,
-    val currentLocation: Location,
-    val currentPrisonStatus: PrisonStatus,
+    val details: String?,
```

Comment:
By digging into the source code of `prisoner-offender-events` I found that these three properties are all nullable
https://github.com/ministryofjustice/prison-offender-events/blob/22dd9fdba858ae227d6facc19195d13921651792/src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/ReleasePrisonerReasonCalculator.kt#L77-L79